tests/test_wave_03.py

```diff
@@ -54,12 +55,15 @@ def test_friends_unique_movies_not_duplicated():
 
     # Arrange
     assert len(friends_unique_movies) == 3
+    assert FANTASY_4 in friends_unique_movies
+    assert INTRIGUE_3 in friends_unique_movies
+    assert {'genre': 'Horror', 'rating': 3.5, 'title': 'It Came from the Stack Trace'} in friends_unique_movies
```

Comment:
I'd recommend using the constant for this one as well:

```suggestion
    assert HORROR_1 in friends_unique_movies
```